changelog.txt

```diff
@@ -1,3 +1,21 @@
+4.8.0.
+============
+Bug fix:
+**Cannot acquire token in UWP app on HoloLens via a unity plugin as json serialization fails** MSAL now uses Newtonsoft.Json for serialization. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1468)
```

Comment:
I would not write it uses Newtonsoft.JSon (as it's really using the internal namespace). Just write that the serialization is now fixed.